.pre-commit-config.yaml

```diff
@@ -71,6 +71,12 @@ repos:
         exclude: ^src/em/emoji.*\.json$
         exclude_types: [yaml]
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Please could you move this before pyproject-fmt? Just because that's where it is in some of my other projects and it makes it easier to diff and compare.

Let's bump the other repos whilst we're here: `pre-commit autoupdate --jobs 0`